Source/WebCore/Modules/WebGPU/GPUQueue.cpp

```diff
@@ -409,7 +409,7 @@ static void getImageBytesFromVideoFrame(WebGPU::Queue& backing, const RefPtr<Vid
 }
 #endif
 
-static void imageBytesForSource(WebGPU::Queue& backing, const auto& sourceDescriptor, const auto& destination, bool& needsYFlip, bool& needsPremultipliedAlpha, ImageDataCallback&& callback)
+static void imageBytesForSource(WebGPU::Queue& backing, const auto& sourceDescriptor, const auto& destination, bool& needsYFlip, bool& needsPremultipliedAlpha, NOESCAPE ImageDataCallback&& callback)
```

Comment:
nit: if callback is indeed called synchronously, I think it'd be nicer to use `NOESCAPE const ImageDataCallback&` since we're not transferring ownership or intending to store the callback somewhere.